src/inmanta/app.py

```diff
@@ -620,6 +621,74 @@ def export(options: argparse.Namespace) -> None:
     summary_reporter.print_summary_and_exit(show_stack_traces=options.errors)
 
 
+def validate_logging_config_parser_config(
+    parser: argparse.ArgumentParser, parent_parsers: abc.Sequence[ArgumentParser]
+) -> None:
+    """
+    Config parser for the validate-logging-config command.
+    """
+    parser.add_argument(
+        "filename",
+        help="Path to the logging configuration file.",
+    )
```

Comment:
shouldn't we use the `--logging-config`

Reply:
Yes, we could. But then we need to document clearly how the command gets its input, because `--help` will otherwise not show any useful information.